client/src/components/Today/Task.tsx

```diff
@@ -32,17 +51,16 @@ export default function Task({ activity, tagsById }: TaskProps) {
 				/>
 				<Checkbox checked={activity.completed} />
 			</S.CheckboxWrapper>
-			<S.TaskName>{activity.name}</S.TaskName>
-			<S.Times>
-				{activityStart(activity).format("HH:mm")}
-				{" - "}
-				{activityEnd(activity).format("HH:mm")}
-			</S.Times>
-			<S.Tags>
+			<T.Times>
+				<span>from {activityStart(activity).format("HH:mm")}</span>
+				<span>to {activityEnd(activity).format("HH:mm")}</span>
```

Comment:
This needs to be different for all-day and/or multiday activities. Leave a note on it in the code so we don't forget to get back to that later.